src/desktop/apps/auction/components/DOM.js

```diff
@@ -27,7 +27,9 @@ class DOM extends Component {
 
     this.$ = require("jquery")
     this.addEventListeners()
-    this.maybeStartRegistrationFlow()
+
+    this.handleRegistrationFlowPath()
+    this.handleConfirmRegistrationPath()
```

Comment:
Maybe...
```jsx
this.maybeRegistrationFlowPath()
this.maybeConfirmRegistrationPath()
```
The "handle" verbiage implies an action in response to something direct, vs a conditional path that may or not happen.